data/en/get_started.json

```diff
@@ -813,6 +925,72 @@
                 ]
               },
               "details": "content/en/get-started/cards/vm-680.md"
+            },
+                        {
+              "id": "vm-673",
+              "item": {
+                "title": "CDAP for VM",
+                "img": "/images/get-started/vm.svg",
+                "buttons": [
+                  {
+                    "buttonTitle": "CDAP 6.7.3 for VM",
+                    "id": "version-673-vm",
+                    "buttonHref": "https://github.com/cdapio/cdap-build/releases/download/v6.7.3/cdap-sandbox-6.7.3.ova"
```

Comment:
I get 404 for  https://github.com/cdapio/cdap-build/releases/download/v6.7.3/cdap-sandbox-6.7.3.ova

Reply:
Discussed offlines, these links are not even used.